### DISCUSSION TITLE
Show event items with same category in Detail.html

### DISCUSSION THREAD

**Discussion by wilbv** [derhansen/sf_event_mgt#1027] [Q&A]
Hi,

Would something like this be possible for sf_event_mgt?
https://docs.typo3.org/p/georgringer/news/main/en-us/singlehtml/Index.html#show-news-items-with-same-category-in-detail-html

I tried this:

```
lib.tx_sfeventmgt.relatedByCategory = USER
lib.tx_sfeventmgt.relatedByCategory {
  userFunc = TYPO3\CMS\Extbase\Core\Bootstrap->run
  extensionName = SfEventMgt
  pluginName = Pievent
  vendorName = DERHANSEN

  switchableControllerActions {
    Event {
      1 = list
    }
  }

  settings < plugin.tx_sfeventmgt.settings
  settings {  
    detailPid = 31
    storagePage = 21
    categoryConjunction = or
    category.current = 1
  }
}
```

This results in php explode. Do you maybe see a quick solution?
Thank you!

**Comment by derhansen** [comment]
Yes, basically the same is possible for ext:sf_event_mgt. Note, that in v6.x of the extension, `switchableControllerActions` are not used any more, so you have to use the `Pieventlist` plugin instead.

**Reply by wilbv** [reply]
We are missing the useStdWrap method of the tx_news snippet though, no?

**Reply by derhansen** [reply]
Yes, `useStdWrap` is not implemented in ext:sf_event_mgt and I will not add this. I suggest you create a custom ViewHelper instead
